app/frontend/src/components/AvatarInput.tsx

```diff
@@ -0,0 +1,29 @@
+import Avatar from "@material-ui/core/Avatar";
+import IconButton from "@material-ui/core/IconButton";
+import React from "react";
+
+import {ImageInput, ImageInputProps} from "./ImageInput";
+
+export interface AvatarInputProps extends ImageInputProps {
+    src?: string;
+    username?: string;
+}
+
+const AvatarInput: React.FC<AvatarInputProps> = ({id, name, username, src, ...rest}) => {
+    return (
+        <ImageInput id={id} name={name} {...rest}>
+            <IconButton component="span">
+                <Avatar
+                    src={src}
+                    imgProps={{style: {objectFit: 'contain'}}}
+                    style={{height: 120, width: 120, objectFit: 'contain'}}
```

Comment:
We tend to use jss (`const useStyle = makeStyles((theme) => ({...}))`) rather than inline styles (unless there is a reason you wanted inline here?)

Also I think it would be good if the parent could set the size (perhaps by this component accepting a `className` prop) rather than it being fixed.

Reply:
I think that's a good idea, I'll change it